unit-tests/func/frame-drops/test-frames-drops.py

```diff
@@ -0,0 +1,164 @@
+# License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Since these 3 tests check for drops while fps latency, maybe this file should be called `test-fps-latency.py`?